Blog.API/Blog.API/Middlewares/GlobalExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SpanJson;
+using System;
+using System.Threading.Tasks;
+
+namespace Blog.API.Middlewares;
+
+public sealed class GlobalExceptionHandlingMiddleware : IMiddleware
```

Comment:
I think it's better to use logger to show and store internal errors. I think it would be better if logger were here. Default logger should be enough for now